Tests/Fuzzers/Makefile

```diff
@@ -1,21 +0,0 @@
-include ../../mk/platform.mk
```

Comment:
We removed the Makefile but didn't add `CMakeLists.txt` instead...

Reply:
Yes you're correct I will check how i can properly integrate Fuzzer and Asan to cmake.